requirements.txt

```diff
@@ -1,31 +1,34 @@
-Django==2.2.13
-dj-database-url==0.5.0
+Django==2.2.21
 wagtail==2.12.4
-psycopg2==2.7.4
-libsass==0.8.3
-Pillow==7.1.2
+dj-database-url==0.5.0
+psycopg2==2.8.6
+libsass==0.20.1
+Pillow==8.1.2
 embedly==0.5.0
-Markdown==2.6.11
-Pygments==2.2.0
+Markdown==3.3.4
+Pygments==2.7.4
 django-compressor==2.2
 django-libsass>=0.7
-raven==6.7.0
-whitenoise==3.3.1
+whitenoise==5.2.0
 brotlipy==0.7.0
 django-storages==1.6.6
 boto3==1.7.82
 django-redis==4.9.0
+sentry-sdk==1.0.0
+scout-apm==2.18.0
+fabric3==1.14.post1
```

Comment:
You've added this to `requirements-dev.txt` as well. Did you mean to add it here in the base requirements?

Reply:
nice catch. borked rebase